src/prog_translator.h

```diff
@@ -12,6 +12,16 @@ namespace cyclus {
 class ExchangeGraph;
 class ExchangeNodeGroup;
 
+/// @brief struct to hold all problem instance state
+struct ProgTranslatorContext {
+  std::vector<double> obj_coeffs;
+  std::vector<double> row_ubs;
+  std::vector<double> row_lbs;
+  std::vector<double> col_ubs;
+  std::vector<double> col_lbs;
+  CoinPackedMatrix m;
+};
+
```

Comment:
@rwcarlsen 
This specific change cause an ABI issues:

> -cyclus::ProgTranslator::Context::Context()
> -cyclus::ProgTranslator::Context::~Context()

I don't really understand the difference before and after the change. Could you explain why you did it ?
thx